staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go

```diff
@@ -200,7 +200,11 @@ func (w *WebhookAuthorizer) Authorize(ctx context.Context, attr authorizer.Attri
 		r.Status = result.Status
 		if shouldCache(attr) {
 			if r.Status.Allowed {
```

Comment:
maybe this is simpler :
```suggestion
			if r.Status.Allowed && !r.Status.Denied
```